packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -290,13 +302,30 @@ export const Tooltip = styled.span< TooltipProps >`
 	) }
 `;
 
+const inputNumberWidth = ( { size }: InputNumberProps ) => {
+	const sizes = {
+		default: space( 16 ),
+		'__unstable-large': space( 20 ),
+	};
+
+	if ( size === '__unstable-large' ) {
+		return css( {
+			width: `${ sizes[ '__unstable-large' ] }`,
+		} );
+	}
+
+	return css`
+		width: ${ sizes.default };
+	`;
+};
```

Comment:
We can simplify the logic in this function a bit:

```suggestion
const inputNumberWidth = ( { size }: InputNumberProps ) => {
	const sizes = {
		default: space( 16 ),
		'__unstable-large': space( 20 ),
	};

	return css`
		width: ${ sizes[ size ?? 'default' ] };
	`;
};
```

Reply:
Good call